src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -5326,6 +5327,13 @@ components:
           uniqueItems: true
           items:
             $ref: '#/components/schemas/KeyUsage'
+        subject_alternative_names:
+          type: string
+          format: text
+          description: certificate subject alternative names
+          example: DNS:*.dev.acme.org
```

Comment:
It is a good practice to use IANA reserved example domain names in documentation (see https://tools.ietf.org/html/rfc2606#section-3) instead of some arbitrary names that can be reserved.  E.g. "example.org" is one such domain.